packs/panther.yml

```diff
@@ -7,9 +7,13 @@ PackDefinition:
     - Panther.SAML.Modified
     - Panther.Sensitive.Role
     - Panther.User.Modified
+    - Confluence.0DayIPs
+    - IOC.SunburstFQDNIOCs
+    - IOC.SunburstSHA256IOCs
     # Data Model
     - Standard.Panther.Audit
     # Helpers
     - panther_base_helpers
     - panther_event_type_helpers
+    - panther_iocs
```

Comment:
if we do not include the rapid-response detections, we should also not include this global helper.